src/containers/paper-canvas.jsx

```diff
@@ -153,6 +153,7 @@ class PaperCanvas extends React.Component {
         } else if (this.shouldZoomToFit === true) {
             zoomToFit(isBitmapMode);
         }
+        this.props.updateViewBounds(paper.view.matrix);
```

Comment:
Has this just always been a bug since launch? I'm surprised we missed that o_o

If this.shouldZoomToFit is false, this function should be a no-op. We only need to updateViewBounds if the conditionals above apply.

Reply:
Yeah I think it has been an issue since launch. 

And good call, I inlined the update in each individual case instead, see fc08bec